runelite-client/src/main/java/net/runelite/client/plugins/skillcalculator/skills/WoodcuttingBonus.java

```diff
@@ -24,16 +24,27 @@
  */
 package net.runelite.client.plugins.skillcalculator.skills;
 
+import java.util.EnumSet;
+import java.util.Set;
 import lombok.AllArgsConstructor;
 import lombok.Getter;
 
 @AllArgsConstructor
 @Getter(onMethod_ = @Override)
 public enum WoodcuttingBonus implements SkillBonus
 {
-	LUMBERJACK_OUTFIT("Lumberjack Outfit (+2.5%)", 1.025f),
+	FELLING_AXE_RATIONS("Felling axe + rations (110%)", 1.1f),
+	LUMBERJACK_OUTFIT("Lumberjack Outfit (102.5%)", 1.025f),
```

Comment:
The easiest way to keep the bonus format consistent is to remove it from the string completely. If all bonuses will be formatted `(100%)`, the string can be put together on the UI side via `100 * getValue()`. Hence, I'd recommend the bonus names become _just the name_ and nothing else.

Reply:
Do I just use 
```java
JLabel uiLabel = new JLabel(bonus.getName() + " (" + (100 * bonus.getValue()) + ")");
```

or do I make a method to generate the display name?